spec/services/search/search_builder_spec.rb

```diff
@@ -81,6 +88,8 @@
         }
       end
 
+      before { allow(Search::BufferSuggestionsBuilder).to receive_message_chain(:new, :buffer_suggestions) }
```

Comment:
I'm not sure what this line (and 113) is achieving?

Reply:
It's about isolating what we're actually testing. We don't want either radius suggestions or buffer suggestions to actually be called, because we don't care about that in these tests - and if they are called we need to be mocking other unrelated things

Reply:
At least that's why I did it in the first place